modules/persistent-storage-rhv.adoc

```diff
@@ -0,0 +1,72 @@
+[id="persistent-storage-rhv_{context}"]
+= Creating a persistent volume on RHV
+
+When you create a `PersistentVolumeClaim` (PVC) object, {product-title} provisions a new persistent volume (PV) and creates a `PersistentVolume` object.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You provided the correct {rh-virtualization} credentials in  `ovirt-credentials` secret.
+* You have installed the oVirt CSI Driver.
+* You have defined at least one storage class.
+
+.Procedure (UI)
+To dynamically create a persistent volume on {rh-virtualization} using the web console:
+
+. In the {product-title} console, click *Storage* -> *Persistent Volume Claims*.
+. In the persistent volume claims overview, click *Create Persistent Volume Claim*.
+. Define the required options on the resulting page.
+. Select the appropriate `StorageClass` (`ovirt-csi-sc` by default).
+. Enter a unique name for the storage claim.
+. Select the access mode, currently RWO (ReadWriteOnce) is the only supported access mode.
+. Define the size of the storage claim.
+. Select the Volume Mode:
++
+`Filesystem` - mounted into pods as a directory (default)
++
+`Block` - block device, without any file system on it
++
+. Click *Create* to create the `PersistentVolumeClaim` and generate a `PersistentVolume`.
+
+.Procedure (CLI)
+
+To dynamically create a {rh-virtualization} CSI volume using the command-line interface (CLI):
+
+. Create and save a file with the `PersistentVolumeClaim` object described by the following YAML:
++
+.pvc-ovirt.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-ovirt
+spec:
+  storageClassName: ovirt-csi-sc <1>
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+    storage: 100Gi  <2>
+  volumeMode: Filesystem | Block <3>
+----
+<1> Name of the required storage class.
+<2> Volume size.
+<3> Supported options:
+* `Filesystem` - mounted into pods as a directory.
+* `Block` - block device, without any file system on it.
++
+. Create the object you saved in the previous step by running the following command:
++
+----
+$ oc create -f pvc-ovirt.yaml
+----
++
+A new PVC is created.
++
+. To verify that the volume was created and is ready, run the following command:
++
+----
+$ oc get pvc pvc-ovirt
+----
++
+The `pvc-ovirt` shows that it is Bound.
```

Comment:
Can a user call this something else? Must it always be a `pvc-ovirt`?

Reply:
No but above when we created the pvc-ovirt.yaml the name we gave is pvc-ovirt.
this is an example

Reply:
It's still not clear from the context that this is an example. If you don't want to clarify here  and at line 41 that this is an example, you need to be clear at line 33 that you're making a PVC with the name `pvc-ovirt`.